use_cases/examples/home_loan_delinquency/deploy/run_dag.py

```diff
@@ -72,16 +73,30 @@ def dbt_job(name, image_name, arguments=[], env_vars={}, repo=REPO):
     }
 ) as dag:
 
-    reporting_job = dbt_job(
-        name='home-loan-delinquency',
+    load_job = dbt_job(
+        name='bq-data-load',
+        image_name='bq-data-load',
+        env_vars={
+            'HOMELOAN_BQ_DATA': 'homeloan_data',
+            'HOMELOAN_BQ_EXPECTEDRESULTS': 'homeloan_expectedreresults',
+            'PROJECT_ID': PROJECT_ID,
+            'GCS_INGEST_BUCKET': GCS_INGEST_BUCKET,
+        }
+    )
+
+    run_hld_report = dbt_job(
+        name='hld-run',
         image_name='home-loan-delinquency',
         arguments=[
+            "run",
+            "--profiles-dir",
```

Comment:
I'd rather it be using the DBT_PROFILES environment variable, as otherwise it is confusing -- the --profiles-dir refers to profiles baked in the container.

Reply:
yes, agree!